server/package.json

```diff
@@ -12,11 +12,13 @@
     "debug": "~2.6.9",
     "dotenv": "^8.2.0",
     "express": "~4.16.1",
+    "googleapis": "39",
```

Comment:
any reason why you are using a version from 2 years ago versus most recent? you are also missing the rest of the version name, example 39.0.0. If this app relies only up to this version, you can use "~39.0.0"
https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json